### PR TITLE
Refactor clickable cards

### PR DESCRIPTION
**Pull request by amersarosong**
## Pull Request Checklist

- [ ] Does this PR includes changes from server side such as workflow, gateway, etc.?
- [x] Have you removed any unnecessary console.log?
- [x] Have you checked all variables, class names, function names, components, file names etc comply with our coding standards? 
- [x] Have you checked any spelling errors/issues?

### CODE REVIEW

File: src/services/event.service.ts [hunk 1]
```diff
@@ -25,15 +25,11 @@ class EventService {
       ? ((response as CommandResponseDto).result as IEvent[]) : response;
   }
 
-async getUpcomingEvents(type: string) {
+async getUpcomingEvents() {
     const response = await ranaoHrsService.Get((api) =>
       api.getRecords('workdays', 'event', [
         JSON.stringify([
           {
-            type: {
```
Comment (mosmdansalan) [inline src/services/event.service.ts:33]: why did you remove this filter? Is this the query to retrieve the "7" Upcoming Events? You may still need the filter here.

Comment (amersarosong) [reply src/services/event.service.ts]: yes sir, I realized my mistake, that filter is really needed, so I have returned it in my recent commits and fixed some bugs.

**Review by mosmdansalan — APPROVED** [review]
